dotcom-rendering/src/model/enhance-numbered-lists.ts

```diff
@@ -82,9 +82,9 @@ const extractStarCount = (element: CAPIElement): number => {
 	return starCount;
 };
 
-const isStarableImage = (element: CAPIElement): boolean => {
+const isStarableImage = (element: CAPIElement | undefined): boolean => {
```

Comment:
This might be `undefined`.

Reply:
What change caused this to flag up, or should it have been flagged before?